hw/system/spatz_cluster/Makefile

```diff
@@ -146,7 +146,8 @@ clean.vsim:
 ## Build compilation script and compile all sources for VCS simulation
 bin/spatz_cluster.vcs: test/bootrom.bin work-vcs/compile.sh work/lib/libfesvr_vcs.a ${TB_DIR}/common_lib.cc test/bootdata.cc test/bootrom.bin test/uartdpi/uartdpi.c
 	mkdir -p bin
-	vcs -Mlib=work-vcs -Mdir=work-vcs -debug_access+all -fgp -kdb +vcs+fsdbon -o bin/spatz_cluster.vcs -j4 -cc $(CC) -cpp $(CXX) \
+	# Default to fast simulation flags, use `-debug_access+all +vcs+fsdbon` for waveform debugging
+	vcs -Mlib=work-vcs -Mdir=work-vcs -O2 -debug_access=r -debug_region=1,tb_bin -kdb -o $@ -j4 -cc $(CC) -cpp $(CXX) \
```

Comment:
Why removing `-fgp`?

Reply:
I did not find this flag in the helper list of VCS (`vcs -h`), so I removed it